python/pyabacus/tests/test_diago_dav_subspace.py

```diff
@@ -0,0 +1 @@
+from __future__ import annotations
```

Comment:
Plz add the tests compared with `spcipy`!

Reply:
i have added tests :)